templates/shortcodes/compare_slider.html

```diff
@@ -0,0 +1,10 @@
+<p class="image-compare-instruction">Drag this image to compare</p>
+<div
+    {% if start_slider_at %}style="--slider-value: {{start_slider_at}}"{% endif %}
```

Comment:
I removed  the `aspect-ratio: 16/9` since it looks like it is defined in the `_image_compare.scss` now.

Reply:
`aspect-ratio` is still required in the general case. The CSS has 16/9 as a default value, but if the images have a different aspect ratio then this must be declared for the widget to behave correctly (I explicitly set the aspect-ratio even on 16/9 images so that future uses have an example that doesn't elide this detail).

Perhaps there's a way for zola to automatically divine the aspect ratio of the images?

Reply:
Fairly certain it is possible, let me add this

Reply:
yeah I've thought about this problem a bit.. I think you could wrap the whole thing in one more div with a maximum height (maybe specified as an optional argument?). Without the short-code, adding yet-another-element to the markup is not great, but I think it's reasonable for the short-code to generate a container element for this purpose. 

I think a div with a max-width or max-height should more or less just be a drop in solution. I haven't tested it though